src/app/modules/main/wallet_section/module.nim

```diff
@@ -40,7 +40,7 @@ type
 proc newModule*[T](
   delegate: T,
   events: EventEmitter,
-  tokenService: token_service.ServiceInterface,
```

Comment:
we don t do interface anymore?

Reply:
It's in the case of an async service, aka a service which has one or more async functions, it can't inherit from the Interface, it needs to inherit from QObject